petab/core.py

```diff
@@ -177,11 +172,6 @@ def get_parameter_df(parameter_file_name):
 
     parameter_df = pd.read_csv(parameter_file_name, sep='\t')
 
-    try:
-        parameter_df.set_index(['parameterId'], inplace=True)
-    except KeyError:
-        raise KeyError('Parameter table missing mandatory field `parameterId`.')
```

Comment:
Why removed?

Reply:
Could the index have caused problems, bc then it does not appear as a column any more, s.t. sometimes one has to do reset_index() before some query?

Reply:
we removed the set_index because the name of that column did not appear (as Yannik mentioned)